zerver/openapi/zulip.yaml

```diff
@@ -5480,6 +5481,1326 @@ paths:
+                          be linkified on matching. The second element is the url with which the
+                          pattern matching string should be linkified with and the third element
+                          is the id of the realm filter.
+                      realm_user_groups:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/UserGroup"
+                        description: |
+                          Present if `realm_user_groups` is present in `event_types`.
+
+                          An array of dictionaries where each dictionary describes a user group.
+                      realm_bots:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/Bot"
+                        description: |
+                          Present if `realm_bot` is present in `event_types`.
+
+                          An array of dictionaries where each dictionary describes a bot owned
+                          by the user.
+                      realm_embedded_bots:
+                        type: array
+                        items:
+                          type: object
+                          additionalProperties: false
+                          description: |
+                            Object containing details of the bot.
+                          properties:
+                            name:
+                              type: string
+                              description: |
+                                The name of the bot.
+                            config:
+                              $ref: "#/components/schemas/Config"
+                        description: |
+                          Present if `realm_embedded_bots` is present in `event_types`.
+
+                          An array of dictionaries where each dictionary describes a embedded bot
+                          of the server.
+                      realm_incoming_webhook_bots:
+                        type: array
+                        items:
+                          type: object
+                          additionalProperties: false
+                          description: |
+                            Object containing details of the bot.
+                          properties:
+                            name:
+                              type: string
+                              description: |
+                                The name of the bot.
+                            config:
+                              $ref: "#/components/schemas/Config"
+                        description: |
+                          Present if `realm_incoming_webhook_bots` is present in `event_types`.
+
+                          An array of dictionaries where each dictionary describes a incoming webhook
+                          bot of the server.
+                      raw_recent_private_conversations:
```

Comment:
This should be `recent_private_conversations`; the `raw_` prefix is an internal implementation detail deleted in `post_process_state`.  There's likely something wrong with how you're doing the testing that you're including this name.

Reply:
Somehow I missed the `post_process_state` function while looking at the mechanism of register.
Luckily it just 4 endpoints so I will change the documentation today itself. I will also change the testing which was initially based on `fetch_initial_test_data`.